aioshelly/rpc_device/device.py

```diff
@@ -498,7 +501,10 @@ def version(self) -> str:
     @property
     def model(self) -> str:
         """Device model."""
-        return cast(str, self.shelly["model"])
+        model = cast(str, self.shelly["model"])
+        if model in SHELLY_X_MODELS:
+            return f"{model}-{self.shelly['jwt']['p']}"
```

Comment:
I don't think we should mix two models from 2 vendors as a single model name.
The model property should return what we get from `shelly["model"].
We can add another property that will return the vendor model, but I don't see the benefit of it as the `jwt` already contains the model name so we are just duplicating data into `SHELLY_X_MODELS`